core/src/main/java/tc/oc/pgm/listeners/AntiGriefListener.java

```diff
@@ -90,12 +90,25 @@ private void participantDefuse(Player player, Entity entity) {
                     "moderation.defuse.player",
                     clicker.getBukkit(),
                     owner.getBukkit().getDisplayName(clicker.getBukkit()) + ChatColor.RED));
+
+        ChatDispatcher.broadcastAdminChatMessage(
+            TranslatableComponent.of(
+                "moderation.defuse.alert.player",
+                TextColor.WHITE,
```

Comment:
```suggestion
                TextColor.GRAY,
```
Consider the gray chat color so formatting is more aligned with other admin broadcasts (e.g /sn) 🙂